DirectEmployers/MyJobs
# [PD-2908] fix v2 mobile nav bug

**Purpose:**
Fix a JavaScript error, see screenshot, which prevented v2 mobile bottom nav from working.   In the file topbar_v2-184-23.js, line #125, I prefixed the readCookie method with the "utils" object literal name.  Also updated the manual static file version numbers. 

<img width="1493" alt="screen shot 2017-01-23 at 2 58 24 pm" src="https://cloud.githubusercontent.com/assets/5124153/22222918/e3d38432-e186-11e6-828b-fce64b9113db.png">

**To Test:**
1.  In v2 template, please switch to mobile view in the browser, ensure console error above isn't there.  Please ensure mobile navigation menus slides up when you click on the nav icons.  (The alignment for the Beta tab might not align correctly due to us moving the NUO option, that fix will be in a future PR.)
